content/faq.ja.md

```diff
@@ -13,6 +13,33 @@ name: よくある質問
 ---
 # よくある質問
 
+<!--#{ using h4s for questions to keep them out of the right side nav (too cluttered when they display) and to provide appropriate text size for questions. #}-->
+#### XRP Ledger(XRPL)はRippleが所有するプライベートブロックチェーンですか？
+
+いいえ。XRPLは分散型のパブリックブロックチェーンです。トランザクション処理やコンセンサスに影響を与えるような変更は、ネットワークバリデーターの80％以上の承認が必要です。Rippleはネットワークへの貢献者の一員ですが、その権利は他の貢献者たちと同じです。ネットワークには150以上のバリデーターが参加しており、うちデフォルトのユニークノードリスト(UNLs)に登録されているものは35以上です。Rippleはこれらのうち6つのノードを運営しています。
+ユニークノードリスト(UNLs)に関しては[ユニークノードリスト（UNL）とは何ですか?](#ユニークノードリストunlとは何ですか)の項目を参照してください。
+
+
+#### プルーフ・オブ・ワーク(PoW)が最善の検証メカニズムではないのですか？
+
+いいえ。プルーフ・オブ・ワークは、信頼できる第三者を必要とせずに二重支出の問題を解決する最初のコンセンサンス・メカニズムでした。[XRP Ledgerのコンセンサス・メカニズム](consensus.html)は、同じ問題をはるかに速く、安く、より良いエネルギー効率で解決します。
+
+
+#### どのようにして持続可能なブロックチェーンを実現するのでしょうか？
+
+ビットコインのエネルギー消費量は、2021年現在アルゼンチンのエネルギー総消費量に相当します。また、ビットコインの採掘者が使用する電力の多くは環境を汚染するリソースから供給されていることが広く報告されています。XRPLは、プルーフ・オブ・ワークのようにエネルギーを浪費しない[コンセンサス・メカニズム](intro-to-consensus.html)を通じてトランザクションを承認し、カーボン・オフセットを活用して、[真にカーボンニュートラルな最初のブロックチェーンのひとつ](https://ripple.com/ripple-press/ripple-leads-sustainability-agenda-to-achieve-carbon-neutrality-by-2030)となっています。[グリーン・カレンシー・カルキュレーター](carbon-calculator.html)を利用して、現金、クレジットカード、その他の人気のある暗号通貨と比較したXRPのエネルギー消費量を調べてみてください。
+
+
+#### XRPLではXRP以外の通貨も取引できますか？
+
+はい。XRPLは米ドル、ユーロ、石油、金、リワードポイントなど、任意の資産をトークン化できるように構築されており、どんな通貨でもXRPL上で発行することができます。成長中のXRPLコミュニティは様々なフィアットトークンや暗号通貨をサポートしており、これを裏付けています。
+
+
+#### XRPLは決済用だけではないのですか？
+
+いいえ。当初は決済のユースケース向けに開発されましたが、台帳であるXRP Ledgerとそのネイティブ・デジタルアセットであるXRPの両方は、さまざまな革新的ブロックチェーンのユースケースで更に人気を集めています。NFT発行のための新しい規約提案、XRPL Labsのスマートコントラクト機能に関するフック修正案、サイドチェーンの開発などが現在進行中です。
```

Comment:
This statement is out of date since NFT is now enabled on the mainnet.

It would need to be corrected in the English version and would be fine in a separate PR.

Reply:
updated English version on #1669